ocs_ci/ocs/platform_nodes.py

```diff
@@ -2167,3 +2167,56 @@ def detach_volume(self, volume, node=None, delete_from_backend=True):
         self.vsphere.remove_disk(
             vm=vm, identifier=volume, key="disk_name", datastore=delete_from_backend
         )
+
+class IBMCloud(NodesBase):
```

Comment:
I think you need to add this class also to hte cls_map on line 65. Only then it can be used during the execution I think.